src/components/EventList/components/EventItem/event-item.scss

```diff
@@ -16,3 +16,7 @@
 .event-item:hover .descriptionText {
   color: $neutral-white;
 }
+
+.event-details {
+  width:100%;
+}
```

Comment:
Did you add this to center the event details?
With width 100%:
<img width="1167" alt="Screen Shot 2021-05-07 at 9 12 39 AM" src="https://user-images.githubusercontent.com/35319956/117454671-62662480-af14-11eb-9553-12d6b8d23710.png">

Without width 100%
<img width="1167" alt="Screen Shot 2021-05-07 at 9 12 56 AM" src="https://user-images.githubusercontent.com/35319956/117454710-6c882300-af14-11eb-97f3-9a55b8c46602.png">

It's a decent update, but wouldn't the better solution be to left-align the text for readability?
<img width="1167" alt="Screen Shot 2021-05-07 at 9 14 15 AM" src="https://user-images.githubusercontent.com/35319956/117454891-9a6d6780-af14-11eb-9c27-7d35b09bfb1c.png">

Also, we should consider getting rid of the "Description:" label. It is pretty redundant, since people are clicking on the event to expand further details.